### ISSUE TITLE
Onderzoeksbank conform layout Overijssel aanpassen

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/geoportaal-test#815]
De layout van de onderzoeksbank wijkt op een aantal punten af van de layout van de provincie Overijssel. Wens is dit conform nieuwe richtlijnen aan te passen, zie plaatje hieronder.

**Comment by gerardnienhuis** [comment]
![onderzoeksbank](https://github.com/IDgis/geoportaal-test/assets/9005444/d60bcfbf-1be7-4035-a8b9-6eff1d08465d)

**Comment by SandroNeumann** [comment]
Wat zijn de verschillen precies? Ik zie het logo in de footer en de naam van de applicatie rechtsonder in de footer. Verder nog?

Impact schatten wanneer dat duidelijk is.

**Comment by gerardnienhuis** [comment]
ja, dat is bijna het enige verschil. Kleuren zijn al wel goed. Merk op: de header is minder hoog dan nu het geval is. Dus:
logo in footer, naam applicatie en header verkorten.
